config/Sage/jobs.txt

```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
 Assistant=-1,-1
 
 Quartermaster=1,1
-Cargo Technician=4,2
-Shaft Miner=3,3
+Cargo Technician=6,2
```

Comment:
I don't think you understand how this config works. The first value is the maximum including latejoins, the second is the maximum roundstart positions.

```suggestion
Cargo Technician=6,4
```